pkg/kv/kvserver/replica_circuit_breaker.go

```diff
@@ -97,13 +96,8 @@ func (br *replicaCircuitBreaker) canEnable() bool {
 	if b {
 		return true // fast path
 	}
-	// IsActive is mildly expensive since it has to unmarshal
-	// a protobuf.
-	if br.st.Version.IsActive(context.Background(), clusterversion.ProbeRequest) {
-		atomic.StoreInt32(&br.versionIsActive, 1)
-		return true
-	}
-	return false // slow path
+	atomic.StoreInt32(&br.versionIsActive, 1)
```

Comment:
If you don'd mind, please remove `br.versionIsActive`, as well as `canEnable()`, since it always returns true.